docs/gatsby-node.ts

```diff
@@ -289,3 +289,12 @@ export const createSchemaCustomization: GatsbyNode["createSchemaCustomization"]
 
   actions.createTypes(typeDefs);
 };
+
+export const onCreateWebpackConfig: GatsbyNode["onCreateWebpackConfig"] = ({ actions }) => {
+  actions.setWebpackConfig({
+    resolve: {
+      // this is necessary for the symlinked packages to be resolved correctly
+      symlinks: false,
```

Comment:
Because of this, we lose Hot Module Replacement, meaning that changing a README.md file in the `@kiwicom/orbit-components` package won't retrigger a build. We need to kill the Gatsby dev server and restart it to see the changes.

I tried a bunch of other things in Gatsby config files to ensure Gatsby MDX would bust its cache when detecting changes in these symlinked files, but it never updated the content even though the server somehow reloaded.

We need proper rendering of the README files, HMR is more a nice-to-have on this docs website IMO.

But that's a tradeoff to be aware of.